cmd/influxd/config.go

```diff
@@ -239,6 +240,10 @@ func NewConfig() *Config {
 	c.Snapshot.BindAddress = DefaultSnapshotBindAddress
 	c.Snapshot.Port = DefaultSnapshotPort
 
+	c.Logging.HTTPAccess = true
```

Comment:
There is an HTTPAPI section.  Should we put logging under that?  This logging section is starting to feel out of place.  Each are could have it's own logging flag now if we wanted.